data/json/items/vehicle_parts.json

```diff
@@ -895,6 +895,20 @@
     },
     {
         "type":"GENERIC",
+        "id" : "hdcamera",
+        "name" : "focused camera",
+        "description" : "A security camera with high quality lenses assembly you could connect to a display.  Allows for greater distance than regular security camera, but the field of vision is limited.",
```

Comment:
"A security camera with an assembly of high quality lenses you could connect to a....." Might be a bit better word flow wise but this is just a suggestion.

Reply:
That indeed sounds better. I'll change that...